### PR TITLE
Remove auth from flippermom when liquidations off

### PR DESCRIPTION
**Pull request by brianmcmichael**
Seems we have to auth flippermom on flip before we can deauth flip on flippermom.

### CODE REVIEW

File: src/DssAction.sol [hunk 1]
```diff
@@ -67,11 +67,16 @@ abstract contract DssAction {
...
+        // Allow FlipperMom to access to the ilk Flipper
         DssExecLib.authorize(co.flip, _flipperMom);
-        // Disallow Cat to kick auctions in ilk Flipper
-        if(!co.isLiquidatable) { DssExecLib.deauthorize(_flipperMom, co.flip); }
+
+        if (!co.isLiquidatable) {
+            // Remove liquidation permission from FlipperMom and Flip
+            DssExecLib.deauthorize(_flipperMom, co.flip);
+            DssExecLib.deauthorize(co.flip, _flipperMom);
+        }
```
Comment (gbalabasquer) [inline src/DssAction.sol:79]: Other option would be:
```
if (!co.isLiquidatable) {
    // Disallow Cat to kick auctions in ilk Flipper
    DssExecLib.deauthorize(co.flip, DssExecLib.cat());
} else {
    DssExecLib.authorize(co.flip, _flipperMom);
}
```

Comment (gbalabasquer) [inline src/DssAction.sol:79]: Other option would be:
```
if (!co.isLiquidatable) {
    // Disallow Cat to kick auctions in ilk Flipper
    DssExecLib.deauthorize(co.flip, DssExecLib.cat());
} else {
    DssExecLib.authorize(co.flip, DssExecLib.flipperMom());
}
```

**Review by gbalabasquer — APPROVED** [review]
LGTM, just a suggestion to make it a bit more efficient (not that explicit though)

File: src/DssAction.sol (hunk 1 above)
Comment (brianmcmichael) [reply src/DssAction.sol]: I like it.

**Review by hexonaut — APPROVED** [review]
LGTM 👍

**Comment by gbalabasquer** [comment]
New changes look good to me.

**Pull request merged**